acceptance/bundle/resource_deps/jobs_update_remote/out.plan_update.direct.json

```diff
@@ -125,6 +124,10 @@
             "action": "skip",
             "reason": "server_side_default"
           },
+          "job_clusters[0].new_cluster.num_workers": {
+            "action": "update",
+            "old": 0
```

Comment:
@denik Should `new` be `null` here?

Reply:
No, absence is expected.

It's untyped nil internally which is correctly rendered as omitted with omitempty.